docs/faq.md

```diff
@@ -576,31 +612,25 @@ plugins {
 }
 ```
 
-### How can I examine network traffic?
+### Why won't my container start?
 
-It can be useful to examine network traffic to diagnose connectivity issues. Jib uses the Google HTTP client library to interact with registries which logs HTTP requests using the JVM-provided `java.util.logging` facilities.  It is very helpful to serialize Jib's actions using the `jib.serialize` property.
+There are some common reasons why containers fail on launch.
 
-To see the HTTP traffic, create a `logging.properties` file with the following:
-```
-handlers = java.util.logging.ConsoleHandler
-java.util.logging.ConsoleHandler.level=ALL
+#### My shell script won't run
+ 
+The default base image used by Jib, ([`distoless/java`](https://github.com/GoogleContainerTools/distroless/tree/master/java)), does not include a shell, and thus shell scripts won't launch.
 
-# CONFIG hides authentication data
-# ALL includes authentication data
-com.google.api.client.http.level=CONFIG
-```
+Solution: use a different base image with a shell.
 
-And then launch your build tool as follows:
-```sh
-mvn -Djava.util.logging.config.file=path/to/log.properties -Djib.serialize=true -Djib.console=plain ...
+#### The container fails with `exec` errors 
+
+A Jib user reported an error launching their container:
 ```
-or
-```sh
-gradle -Djava.util.logging.config.file=path/to/log.properties -Djib.serialize=true -Djib.console=plain ...
+standard_init_linux.go:211 exec user process caused "no such file or directory"
 ```
 
-### How do I view debug logs for Jib?
+On examining the container structure with [Dive](https://github.com/wagoodman/dive), the user discovered that the contents of the `/lib` directory had disappeared.
 
-Maven: use `mvn -X -Djib.serialize=true` to enable more detailed logging and serialize Jib's actions.
+The user had used Jib's ability to install extra files into the image ([Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#adding-arbitrary-files-to-the-image), [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#adding-arbitrary-files-to-the-image)) to install a DLL by placing it in `src/main/jib/lib/file.dll`. This would normally cause the `file.dll` to be installed in the image as `/lib/file.dll`. But the user's base image used symbolic links for `/lib` and `/lib64`. Jib does not follow such symbolic links when creating the image. And at container initialization time, Docker treats these symlinks as a file, and thus the symbolic link was replaced with `/lib`. As a result, none of the system shared libraries were resolved and dynamically-linked programs failed.
```

Comment:
```suggestion
The user had used Jib's ability to install extra files into the image ([Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#adding-arbitrary-files-to-the-image), [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#adding-arbitrary-files-to-the-image)) to install a library file by placing it in `src/main/jib/lib/libfoo.so`. This would normally cause the `libfoo.so` to be installed in the image as `/lib/libfoo.so`. But `/lib` and `/lib64` in the user's base image were symbolic links. Jib does not follow such symbolic links when creating the image. And at container initialization time, Docker treats these symlinks as a file, and thus the symbolic link was replaced with `/lib` as a new directory. As a result, none of the system shared libraries were resolved and dynamically-linked programs failed.
```